src/main/java/org/spongepowered/common/interfaces/IMixinWorld.java

```diff
@@ -100,4 +103,12 @@
 
     void notifyBlockOfStateChange(BlockPos notifyPos, Block sourceBlock, BlockPos sourcePos);
 
+    @Nullable
+    EntityPlayer getClosestPlayerToEntityWhoAffectsSpawning(net.minecraft.entity.Entity entity, double distance);
+
+    @Nullable
+    EntityPlayer getClosestPlayerWhoAffectsSpawning(double x, double y, double z, double distance);
+
+    boolean isAnyPlayerWithinRangeAtWhoAffectsSpawning(double x, double y, double z, double range);
```

Comment:
Okay, I understand what you're trying to get at. The range's position is the `At`, the players affecting spawning are the `Who`.
Not a fan of the name though. I'd suggest something along the lines of `isPlayerNearRangeAffectingSpawning` but even that's pretty long winded. Might just have to go with a short thing and explain in javadocs.